cmd/cmd.go

```diff
@@ -908,8 +902,27 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func versionHandler(cmd *cobra.Command, _ []string) {
+	fmt.Printf("ollama version %s\n", version.Version)
```

Comment:
This should be the host version and not the client version although we should still call it the "Ollama version" since it's the main thing.

Reply:
The client version is the main thing since the server may not be available

Reply:
Wouldn't we tell you that you can't connect to Ollama in that case? It seems like the version is somewhat beside the point.

Reply:
Those are bad examples and "client" is not the right term. It's the version of the `ollama` command.

If the server doesn't start and you want to know why, a first step would be to identify the version.

Similarly if some other subcommand isn't able to connect to the server, a first step would be to get the version and try to reproduce. Whether it's connectivity related or not is irrelevant.

The version of the command is valuable by itself and should be the main focus.

Reply:
I disagree here. The version of the server running the LLM is the key because it can lead to issues not being able to run the model correctly. That's typically where people are going to run into problems. The client (command?) mismatch is also a problem, but that's accounted for by showing the warning.

If we also want to show the client version for cases where you can't connect, I think I'm OK with a similar string to "You client is using version ...", but again, it shouldn't be the main focus.